poucave/html/js/app.js

```diff
@@ -292,6 +309,8 @@ class SystemDiagram extends Component {
       return;
     }
 
+    this.initializeDiagram();
+
```

Comment:
Why do we need it both on after load and update? (deserves a comment if relevant)